src/test/java/webdriverFramework/SSMission.java

```diff
@@ -1,53 +1,211 @@
+import org.openqa.selenium.support.ui.*;
+import pageObjects.*;
 
 import java.io.File;
+import java.util.*;
+
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
 
-/**
- * Created by roman.pipchenko on 1/20/2017.
- */
 public class SSMission {
-    public static void main(String[] args) {
 
-        File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
-        if (file.exists()) {
-            System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
-        }
+    static WebDriver driver;
+
+    public static String browser = "chrome";
 
-        System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
-        WebDriver driver = new FirefoxDriver();
 
+    @Rule
+    public TestRule screenshotTaker = new ScreenshotTaker((TakesScreenshot) driver);
+
+    @BeforeClass
+    public static void setup() {
+        if (browser.equals("chrome")) {
+
+            File file = new File("C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\chromedriver.exe");
+                driver = new ChromeDriver();
+            }
+        } else {
+            File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
+            }
+            System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
+            driver = new FirefoxDriver();
+        }
+    }
 
-        driver.get("https://www.ss.lv");
-        driver.findElement(By.xpath("//*[@href='/ru/']")).click();
+    @Test
+    public void testSS() throws InterruptedException {
         WebDriverWait wait = new WebDriverWait(driver, 5);
+        driver.get("https://www.ss.lv/lv");
+        driver.manage().window().maximize();
+
+        HomePage homePage = new HomePage(driver);
+
+        ElectronicsPage electronicsPage = new ElectronicsPage(driver);
+
+        homePage.switchLangTo(Language.RU);
+
+        wait.until(ExpectedConditions.elementToBeClickable(homePage.goToElectronicsLink));
+
+        homePage.goToElectronicsLink.click();
+
+        wait.until(ExpectedConditions.elementToBeClickable(electronicsPage.searchElectronicsLink));
+
+        electronicsPage.searchElectronicsLink.click();
+
+        SearchElectronicsPage searchElectronicsPage = new SearchElectronicsPage(driver);
 
-        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@class=\"a1\"][@href=\"/ru/electronics/\"]")));
+        wait.until(ExpectedConditions.elementToBeClickable(searchElectronicsPage.searchTextBox));
+
+        searchElectronicsPage.searchTextBox.sendKeys("Laptop");
+        Thread.sleep(1000);
```

Comment:
Really a need to use Thread.sleep? This force you to deal with exception handling. I see FluentWait class, it could be much better..